build/images/base-windows/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG GO_VERSION
+FROM mcr.microsoft.com/windows/servercore:1809 as windows-utility-base
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV CNI_PLUGINS="./host-local.exe"
+
+WORKDIR /
+
+RUN mkdir -Force C:\opt\cni\bin
+
+RUN curl.exe -LO https://github.com/containernetworking/plugins/releases/download/v0.8.1/cni-plugins-windows-amd64-v0.8.1.tgz; \
+    tar -xzf cni-plugins-windows-amd64-v0.8.1.tgz  -C C:\opt\cni\bin ${env:CNI_PLUGINS}; \
+    rm cni-plugins-windows-amd64-v0.8.1.tgz
+
+# Install 7zip, git-for-windows, mingw64 to support "make tool"
+RUN curl.exe -LO https://www.7-zip.org/a/7z2107-x64.exe; \
+    cmd /c start /wait 7z2107-x64.exe /S; \
+    del 7z2107-x64.exe;  $env:Path = $env:Path+';C:/Program Files/7-Zip'; \
+    curl.exe -Lo mingw.7z https://cfhcable.dl.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/seh/x86_64-8.1.0-release-posix-seh-rt_v6-rev0.7z; \
+    7z x mingw.7z; cp c:/mingw64/bin/mingw32-make.exe c:/mingw64/bin/make.exe; \
+    curl.exe -Lo git.exe https://github.com/git-for-windows/git/releases/download/v2.35.1.windows.2/PortableGit-2.35.1.2-64-bit.7z.exe; \
+    7z x git.exe -oC:\git; \
+    mkdir C:\wins; \
+    curl.exe -Lo C:/wins/wins.exe https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
+
+FROM mcr.microsoft.com/windows/nanoserver:1809
+
+SHELL ["cmd", "/S", "/C"]
+
+ENV GOPATH="C:\go"
+
+USER ContainerAdministrator
+
+COPY --from=windows-utility-base ["C:\\\\opt\\\\cni\\\\bin","C:\\\\opt\\\\cni\\\\bin"]
+COPY --from=windows-utility-base ["C:\\\\Program Files\\\\7-Zip","C:\\\\Program Files\\\\7-Zip"]
+COPY --from=windows-utility-base ["C:\\\\git","C:\\\\git"]
+COPY --from=windows-utility-base ["C:\\\\mingw64","C:\\\\mingw64"]
+COPY --from=windows-utility-base ["C:\\\\wins","C:\\\\wins"]
+
+RUN setx /m PATH "C:\Program Files\7-Zip;C:\git\bin;C:\git\usr\bin;C:\mingw64\bin;C:\Program Files\Go\bin;%GOPATH\bin%;%PATH%"
+
+WORKDIR $GOPATH
```

Comment:
do you know what the `GOPATH` is used for here?

I also don't think it is useful to set the `WORKDIR` in the base image. Do you know if it is used for anything?